cmd/skopeo/utils_test.go

```diff
@@ -180,3 +181,50 @@ func TestImageDestOptionsNewSystemContext(t *testing.T) {
 	_, err = opts.newSystemContext()
 	assert.Error(t, err)
 }
+
+// since there is a shared authfile image option and a non-shared (prefixed) one, make sure the override logic
+// works correctly.
+func TestImageOptionsAuthfileOverride(t *testing.T) {
+	// if authfile and dest-authfile is provided, dest-authfile wins
+	{
+		// Set everything to non-default values.
```

Comment:
This comment does not match the data (in none of the three instances), just drop it.

Reply:
refactored test according to your suggestions below.